test/core/modules/benchmark.js

```diff
@@ -30,7 +30,7 @@ test('benchmark all modules', function(t) {
     if (mods.length > 1) { //Last one is test module, we need not benchmark it
       sequencer.steps[global.idx].output.src = image;
       global.idx++;
-      if (mods[0] === 'import-image') {
+      if (mods[0] === 'import-image' || mods[0] === 'fisheye-gl') {
```

Comment:
What do you think about making a module property like "only: ["gl"]" or something rather than using module names?

Reply:
Yeah, I used the `requires` property in the info.json!